sshd/issue_invite.sh

```diff
@@ -72,6 +72,9 @@ cat $TMP/id_rsa.pub >> $HOMEDIR/.ssh/authorized_keys
 PUBLIC_IP=`cat /hostname`
 export CLOUD_INSTANCE_DETAILS="{\"host\":\"$PUBLIC_IP\",\"user\":\"$USERNAME\",\"key\":\"$ENCODED_KEY\"}"
 
+# Output invite in JSON format for the
+echo "ISSUE_INVITE_RESULT:{\"host\":\"$PUBLIC_IP\",\"user\":\"$USERNAME\",\"key\":\"$ENCODED_KEY\"}"
```

Comment:
I think you can do:
```bash
echo "ISSUE_INVITE_RESULT: $CLOUD_INSTANCE_DETAILS"
```

Also, `ISSUE_INVITE_RESULT` is not quite right...how about `CLOUD_INSTANCE_DETAILS_JSON`?